src/components/NcCounterBubble/NcCounterBubble.vue

```diff
@@ -5,34 +5,184 @@
 
  <docs>
 
-### Normal Counter
+### Default usage
 
-```
-<NcCounterBubble>314+</NcCounterBubble>
-```
+NcCounterBubble displays a number from the `count` prop in a bubble.
 
-### Outlined Counter (e.g team mentions)
+By default, the number is **humanized** according to Nextcloud user's locale setting. Humanization can be disabled via `raw` prop.
 
-```
-<NcCounterBubble type="outlined">314+</NcCounterBubble>
+```vue
+<template>
+	<table>
+		<tr>
+			<th>count</th>
+			<th>default</th>
+			<th>raw</th>
+		</tr>
+		<tr v-for="num in numbers" :key="num">
+			<td>{{ num }}</td>
+			<td>
+				<NcCounterBubble :count="num" />
+			</td>
+			<td>
+				<NcCounterBubble :count="num" raw />
+			</td>
+		</tr>
+	</table>
+</template>
+
+<script>
+export default {
+	setup() {
+		return {
+			numbers: [1, 9, 75, 450, 1042, 1750, 1999, 14567, 14567890, 2000000008],
+		}
+	},
+}
+</script>
+
+<style scoped>
+table {
+	border-collapse: collapse;
+}
+
+th,
+td {
+	border: 1px solid var(--color-border);
+	padding: var(--default-grid-baseline) calc(var(--default-grid-baseline) * 2);
+}
+
+th {
+	color: var(--color-text-maxcontrast);
+}
+</style>
 ```
 
-### Highlighted Counter (e.g direct mentions)
+### Styles
+
+Use different styles for different types of counters.
 
 ```
-<NcCounterBubble type="highlighted">314+</NcCounterBubble>
+<template>
+	<table>
+		<tr>
+			<th>type</th>
+			<th>counter</th>
+			<th>Usage example</th>
+		</tr>
+		<tr>
+			<td>'' (default)</td>
+			<td>
+				<NcCounterBubble :count="3" />
+			</td>
+			<td></td>
+		</tr>
+		<tr>
+			<td>outlined</td>
+			<td><NcCounterBubble :count="3" type="outlined" /></td>
+			<td>Team/group mentions</td>
+		</tr>
+		<tr>
+			<td>highlighted</td>
+			<td>
+				<NcCounterBubble :count="3" type="highlighted" />
+			</td>
+			<td>Direct mentions</td>
+		</tr>
+		<tr>
+			<td>outlined active</td>
+			<td class="active-like">
+				<NcCounterBubble :count="3" type="outlined" active />
+			</td>
+			<td>Same as "outlined", but in an "active" container</td>
+		</tr>
+		<tr>
+			<td>highlighted active</td>
+			<td class="active-like">
+				<NcCounterBubble :count="3" type="highlighted" active />
+			</td>
+			<td>Same as "highlighted", but in an "active" container</td>
+		</tr>
+	</table>
+</template>
+
+<style scoped>
+table {
+	border-collapse: collapse;
+}
+
+th,
+td {
+	border: 1px solid var(--color-border);
+	padding: var(--default-grid-baseline) calc(var(--default-grid-baseline) * 2);
+
+	&.active-like {
+		background-color: var(--color-primary-element);
+	}
+}
+
+th {
+	color: var(--color-text-maxcontrast);
+}
+</style>
 ```
 
-</docs>
+### Custom content
+
+You can use slot to pass any custom content.
 
+Note, if you pass a plain number to the default slot, without raw prop it will be humanized like via `count` prop. This feature is **deprecated** and will be removed in the future. Prefer using `count` prop for numbers with humanization.
```

Comment:
Shall we deprecate the whole default slot?
As the counter function is done by the prop, using the slot without the counter handling feels like you want to use NcChip or NcUserBubble.

Reply:
Done
![image](https://github.com/user-attachments/assets/77c8c9dd-421a-4c1e-b9f6-95a3c10399bc)

Reply:
BTW just for my reasoning:
From my point of view it makes no sense that we have so many components supporting the same use case.
This one is specialized for *counting* / showing a *count*, so we should stick with that behavior.

If there are complains about this, we still could add a `unit` prop where you can put `n('row', 'rows')` and the result would be `1 row`, `99 rows`, `1k rows`. But lets see if this is needed.